ouroboros-network/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 * Fix diffusion tests.
 * `demo-chain-sync`: added option parser, added new options.
 
+* Increase keyed timeout on a testnet test that was failing.
+
```

Comment:
Do we want to track test changes in `CHANGELOG`? I am not sure if it's worth it.

Reply:
It was just to shut CI check up 😄